astropy/time/formats.py

```diff
@@ -2122,7 +2123,7 @@ class TimeDeltaSec(TimeDeltaNumeric):
     unit = 1.0 / erfa.DAYSEC  # for quantity input
 
 
-class TimeDeltaJD(TimeDeltaNumeric):
+class TimeDeltaJD(TimeDeltaNumeric, TimeUnique):
```

Comment:
This is not unique, right? I don't see how one can distinguish from `TimeDeltaSec`. Or is it that currently we treat the default as "day"?

Reply:
Yes, initializing `TimeDelta` with a plain number uniquely identifies that as JD because of the default as day. This works even for a string that represents a number e.g. `TimeDelta('12')`.

Reply:
Would still be nice to deprecate that... But probably far too late!